app/controllers/podcast_planner_controller.rb

```diff
@@ -1,10 +1,12 @@
 class PodcastPlannerController < ApplicationController
   before_action :set_podcast
+  before_action :set_beginning_of_week
 
   # GET /podcasts/1/planner
   def show
     @planner = PodcastPlanner.new(planner_params)
     @planner.generate_dates!
+    @draft_dates = @podcast.episodes.draft.pluck(:released_at)
```

Comment:
a new feature i've implemented, to show drafts that already exist for the podcast. these will show in yellow on the calendar.

Reply:
Nice!